### PR TITLE
Setup of production server #47

### PR DESCRIPTION
**Pull request by jalal**
For the production server, I've had to rearrange some of the files and add some setup for the uWSCGI module.
None of the functional logic should have changed, although still requires checking.

Live server is at: http://triffids.gnomedia.net/

It would be good to get this merged in to `master` soon so that the live server can be running on `master`

### CODE REVIEW

File: server/triffidsapi/api.py [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
 
 
-@app.errorhandler(404)
+@api.errorhandler(404)
 def not_found(error):
     return make_response(jsonify({'error': 'Not found'}), 404)
 
 
-@app.route('/')
+@api.route('/')
```
Comment (tomwieck) [inline server/triffidsapi/api.py]: Is this required or should we just remove it / replace it with something?

File: server/triffidsapi/api.py [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
 
-@app.route('/api/v1/parks', methods=['GET'])
+@api.route('/parks', methods=['GET'])
 def getAllParkNames():
-    page = request.args.get('page')
+    page = request.args.get('page') or 1
     response = parks.getAllParkNames(int(page))
     if len(response) == 0:
-        abort(404)
+        abort(402)
```
Comment (tomwieck) [inline server/triffidsapi/api.py]: 402 doesn't seem correct here, think 404 should be fine (or 400?)

File: server/triffidsapi/app.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- create a Flask app instance
+"""
+
+from flask import Flask
+from flask_cors import CORS
+
+
+def create_app(app_name='TRIFFIDS_API'):
+    app = Flask(app_name)
+    # app.config.from_object('surveyapi.config.BaseConfig')
```
Comment (tomwieck) [inline server/triffidsapi/app.py]: can this just be removed?

**Comment by tomwieck** [comment]
Nice work, think this covers this issue too so will move that into complete once this is merged too (unless you have any objections)
https://github.com/tomwieck/triffids/issues/16

File: server/triffidsapi/api.py (hunk 1 above)
Comment (jalal) [reply server/triffidsapi/api.py]: Without the decorators, the routes will not work, AFAIK.

**Pull request merged**